### ISSUE TITLE
Update to Spigot 1.13.2

### ISSUE THREAD

**Issue by RubenBarrionuevo** [HYPExMon5ter/PlayerMention-Recoded#5]
I would like to know if you intend to update the plugin to version 1.13.2 of spigot

The plugin is quite useful, especially for administrators and moderators of the server when we are busy outside the game and so users can tell us.

**Comment by HYPExMon5ter** [comment]
Yes, not sure when I’ll update it, but it’ll be updated when I have time and motivation to do so. Might try to get it done this weekend.

**Comment by blackshot** [comment]
so, what happened? did you update it?

**Comment by HYPExMon5ter** [comment]
I have not, haven’t had the time, been working full time as well as going to school full time, I’m graduating soon though and changing my availability at work so I’m not working 24/7. So hopefully an update soon
